src/components/dialogs/StartPrintDialogSpoolman.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+    <div>
+        <v-divider class="mt-3 mb-0" />
+        <v-card-text class="py-0 px-2">
+            <spoolman-panel-active-spool
+                v-if="activeSpoolId !== null"
+                :small="true"
+                class="my-0"
+                @change-spool="showChangeSpoolDialog = true" />
+            <v-alert v-for="alert in alerts" :key="alert.text" text :color="alert.color" class="mt-4 mx-3">
+                {{ alert.text }}
+            </v-alert>
+            <div class="text-center">
+                <v-btn color="primary" small class="mx-auto" @click="showChangeSpoolDialog = true">
+                    {{ buttonText }}
+                </v-btn>
+            </div>
+        </v-card-text>
+        <v-divider :class="classSecondDivider" />
+        <spoolman-change-spool-dialog :show-dialog="showChangeSpoolDialog" @close="showChangeSpoolDialog = false" />
+    </div>
+</template>
+
+<script lang="ts">
+import { Component, Mixins, Prop } from 'vue-property-decorator'
+import BaseMixin from '@/components/mixins/base'
+import SpoolmanPanelActiveSpool from '@/components/panels/Spoolman/SpoolmanPanelActiveSpool.vue'
+import { FileStateGcodefile } from '@/store/files/types'
+
+@Component({
+    components: { SpoolmanPanelActiveSpool },
+})
+export default class StartPrintDialogSpoolman extends Mixins(BaseMixin) {
+    @Prop({ required: true }) readonly file!: FileStateGcodefile
+
+    showChangeSpoolDialog = false
+
+    get activeSpoolId() {
+        let spoolId = this.$store.state.server.spoolman?.active_spool_id ?? null
+        if (spoolId === 0) spoolId = null
+
+        return spoolId
+    }
+
+    get activeSpool() {
+        return this.$store.state.server.spoolman?.active_spool ?? null
+    }
+
+    get classSecondDivider() {
+        const classes = ['mt-4']
+
+        classes.push(this.moonrakerComponents.includes('spoolman') ? 'mb-2' : 'mb-0')
```

Comment:
not really sure here but... this component should not be loaded if spoolman is not present right? So this condtion will always be true.

Reply:
ok this should be timelapse, because the margin bottom should be different, when timelapse is also loded